lib/activerecord-import/import.rb

```diff
@@ -275,7 +275,11 @@ def import_helper( *args )
           # this next line breaks sqlite.so with a segmentation fault
           # if model.new_record? || options[:on_duplicate_key_update]
             column_names.map do |name|
-              model.read_attribute_before_type_cast(name.to_s)
+              if model.class.column_defaults[name.to_s].is_a? Integer
+                model.read_attribute(name.to_s)
+              else
+                model.read_attribute_before_type_cast(name.to_s)
```

Comment:
Ah ok, that makes sense. It looks like the method `values_sql_for_columns_and_attributes` ultimately handles casting attributes to db safe values now? I think it just using  `read_attribute` might be ok.

@aquajach would you be able to try that out?